frontend/packages/component-library/src/textField/textfield.module.scss

```
@@ -5,225 +5,118 @@
-      color: var(--text-neutral-disabled);
-    }
-
-    .textFieldHelperSection,
-    .textFieldErrorSection {
-      color: var(--text-neutral-disabled);
-    }
+  &:disabled {
+    border-color: var(--borders-neutral-disabled);
+    color: var(--text-neutral-disabled);
   }
 
   // Read Only styles
-  &:has(input:read-only) {
-    input {
-      border-color: var(--borders-neutral-disabled);
-      background-color: var(--background-neutral-secondary);
-    }
+  &:read-only {
+    border-color: var(--borders-neutral-disabled);
+    background-color: var(--background-neutral-secondary);
   }
 }
 
 .textField.textField-gray {
-  input {
-    border-color: var(--borders-neutral-strong);
+  border-color: var(--borders-neutral-strong);
 
-    &.hasError {
-      border-color: var(--borders-error-primary);
-    }
+  &.hasError {
+    border-color: var(--borders-error-primary);
+  }
 
-    &:focus-visible {
-      border-color: var(--borders-neutral-strong);
-    }
+  &:focus-visible {
+    border-color: var(--borders-neutral-strong);
   }
 }
 
 .textField.textField-white {
-  .textFieldLabel {
-    color: var(--text-neutral-inverse);
-  }
-
-  input {
-    border-color: var(--neutral-base-white);
-    background: var(--background-neutral-primary-inverse);
-    color: var(--text-neutral-inverse);
-
-    &::placeholder {
-      color: var(--text-neutral-placeholder);
-    }
+  border-color: var(--neutral-base-white);
+  background: var(--background-neutral-primary-inverse);
+  color: var(--text-neutral-inverse);
 
-    &:focus-visible {
-      border-color: var(--neutral-base-white);
-    }
+  &::placeholder {
+    color: var(--text-neutral-placeholder);
   }
 
-  .textFieldHelperSection,
-  .textFieldErrorSection {
-    color: var(--text-neutral-inverse);
+  &:focus-visible {
+    border-color: var(--neutral-base-white);
   }
 
   // Disabled styles
-  &:has(input:disabled) {
-    .textFieldLabel {
-      color: var(--text-neutral-tertiary);
-    }
-
-    input {
-      border-color: var(--neutral-gray-80);
-      color: var(--text-neutral-tertiary);
-    }
-
-    .textFieldHelperSection,
-    .textFieldErrorSection {
-      color: var(--text-neutral-tertiary);
-    }
+  &:disabled {
+    border-color: var(--neutral-gray-80);
+    color: var(--text-neutral-tertiary);
   }
 
   // Read Only styles
-  &:has(input:read-only) {
-    input {
-      border-color: var(--neutral-gray-70);
-      background-color: var(--neutral-gray-90);
-    }
+  &:read-only {
+    border-color: var(--neutral-gray-70);
+    background-color: var(--neutral-gray-90);
   }
 }
 
 // TextField sizes
 .textField-l {
-  .textFieldLabel {
-    @include label-one;
-    margin-bottom: 0;
-  }
-
-  input {
-    @include body-one;
-    padding: 0.625rem 0.875rem;
-    margin-bottom: 0;
-  }
-
-  .textFieldHelperSection {
-    @include body-two;
-    margin-bottom: 0;
-    gap: 0.375rem;
-  }
+  @include body-one;
+  padding: 0.625rem 0.875rem;
+  margin-bottom: 0;
```

Related to the similar comment above:
```suggestion
.textField {
  &-l {
    @include body-one;
    padding: 0.625rem 0.875rem;
    margin-bottom: 0;
  }
  
  &-m {
    @include body-two;
    padding: 0.5rem 0.75rem;
    margin-bottom: 0;
  } 
```
etc…